azure-functions/acbs-function/api.api-test.js

```diff
@@ -7,65 +7,73 @@ const {
 } = require('./api');
 
 describe('get function', () => {
-  // Tests that the function returns an API response object when endpoint is provided
-  it('should test happy path returns api response object', async () => {
-    const response = await getFacility('1');
-    expect(response).toHaveProperty('status');
-    expect(response).toHaveProperty('data');
+  describe('happy path', () => {
+    it('should test returns api response object', async () => {
```

Comment:
I'm not sure if I'm misunderstanding the purpose of these tests, but is the idea that we are verifying that for both valid and invalid, GET, POST, PUT and PATCH requests, we receive an api response object?

If so, `valid request` and `invalid request` might be a bit clearer than `happy path` and `unhappy path`. 

I think the test names should just be something like `it('returns an api response object...` or `it('should return an api response object...`. We shouldn't need to say that the test should be testing something. The 'it' should describe the production code's behaviour rather than the test code's behaviour if that makes sense.

But I could be misunderstanding what these are testing!